platform/functionality/field_capabilities/field_caps.go

```diff
@@ -47,6 +48,17 @@ func handleFieldCapsIndex(cfg map[string]config.IndexConfiguration, schemaRegist
 
 	schemas := schemaRegistry.AllSchemas()
 
+	var filteredIndexes []string
```

Comment:
Is it better place for that ? https://github.com/QuesmaOrg/quesma/blob/main/platform/clickhouse/clickhouse.go#L156
`clickhouse.ResolveIndexPattern` should filter out `common_table` IMO